.github/workflows/test-application.yaml

```diff
@@ -253,8 +253,8 @@ jobs:
             - name: Install and configure PHP
               uses: shivammathur/setup-php@v2
               with:
-                  php-version: '8.1'
-                  extensions: 'ctype, iconv, mysql, imagick'
+                  php-version: '8.3'
+                  extensions: 'ctype, iconv, mysql'
```

Comment:
be consistent to 2.6 branch for the baseline: https://github.com/sulu/sulu/pull/7306